src/UnitTest.h

```diff
@@ -59,6 +59,12 @@
 ! ASSERT_APPROXEQA used for reals where equality is a comparison to defined epsilon 
 #define ASSERT_APPROXEQA(b,a,msg) ASSERT(b.APPROXEQA.a,msg); FINFO()b,"/=",a
 
+! ASSERT_APPROXEQDS used for reals where equality is a comparison to single precision 
```

Comment:
Based on suggestion of removing these approxeq's, I also suggest re not add these.